From d40ff11727955e77461c7c8ab876efd425cc03cd Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sat, 28 Mar 2020 23:40:02 +0200
Subject: [PATCH 105/113] fix dataUsage informer derp

	modified:   packages/SystemUI/res/layout/quick_status_bar_header_system_icons.xml
	modified:   packages/SystemUI/src/com/android/systemui/statusbar/info/DataUsageView.java
	modified:   packages/SystemUI/src/com/android/systemui/statusbar/phone/StatusBar.java
---
 .../quick_status_bar_header_system_icons.xml  | 26 ++++++++++---------
 .../statusbar/info/DataUsageView.java         |  4 +--
 .../systemui/statusbar/phone/StatusBar.java   | 11 ++++----
 3 files changed, 21 insertions(+), 20 deletions(-)

diff --git a/packages/SystemUI/res/layout/quick_status_bar_header_system_icons.xml b/packages/SystemUI/res/layout/quick_status_bar_header_system_icons.xml
index f13cd550527..b61bf3783e7 100644
--- a/packages/SystemUI/res/layout/quick_status_bar_header_system_icons.xml
+++ b/packages/SystemUI/res/layout/quick_status_bar_header_system_icons.xml
@@ -43,30 +43,32 @@
         android:minWidth="48dp"
         android:gravity="center_vertical|start"
         android:paddingStart="@dimen/status_bar_left_clock_starting_padding"
-        android:paddingEnd="@dimen/status_bar_left_clock_end_padding"
+        android:paddingEnd="2dp"
         android:singleLine="true"
         android:textAppearance="@style/TextAppearance.StatusBar.Clock"
         systemui:showDark="false" />
-    </LinearLayout>
-
-    <android.widget.Space
-        android:id="@+id/space"
-        android:layout_width="0dp"
-        android:layout_height="match_parent"
-        android:layout_gravity="center_vertical|center_horizontal"
-        android:visibility="gone" />
 
     <com.android.systemui.statusbar.info.DataUsageView
         android:id="@+id/data_sim_usage"
         android:layout_width="wrap_content"
         android:layout_height="match_parent"
-        android:gravity="center_vertical|end"
-        android:paddingStart="@dimen/status_bar_left_clock_starting_padding"
-        android:paddingEnd="@dimen/status_bar_left_clock_end_padding"
+        android:gravity="center_vertical|start"
+        android:paddingStart="2dp"
+        android:paddingEnd="0dp"
         android:singleLine="true"
         android:textAppearance="@style/TextAppearance.StatusBar.Clock"
+        systemui:showDark="false"
         android:text="" />
 
+    </LinearLayout>
+
+    <android.widget.Space
+        android:id="@+id/space"
+        android:layout_width="0dp"
+        android:layout_height="match_parent"
+        android:layout_gravity="center_vertical|center_horizontal"
+        android:visibility="gone" />
+
     <LinearLayout
         android:layout_width="0dp"
         android:layout_height="match_parent"
diff --git a/packages/SystemUI/src/com/android/systemui/statusbar/info/DataUsageView.java b/packages/SystemUI/src/com/android/systemui/statusbar/info/DataUsageView.java
index 909fb076c75..bb451dd7dac 100644
--- a/packages/SystemUI/src/com/android/systemui/statusbar/info/DataUsageView.java
+++ b/packages/SystemUI/src/com/android/systemui/statusbar/info/DataUsageView.java
@@ -64,7 +64,7 @@ public class DataUsageView extends TextView {
             SubscriptionManager.getDefaultDataSubscriptionId());
         final DataUsageController.DataUsageInfo info = mobileDataController.getDataUsageInfo();
         
-        formatedinfo = formatDataUsage(info.usageLevel) + " " + mContext.getResources().getString(R.string.usage_data);
+        formatedinfo = formatDataUsage(info.usageLevel) + " ";
         shouldUpdateDataTextView = true;
     }
 
@@ -83,4 +83,4 @@ public class DataUsageView extends TextView {
         return BidiFormatter.getInstance().unicodeWrap(mContext.getString(
                 com.android.internal.R.string.fileSizeSuffix, res.value, res.units));
     }
-}
\ No newline at end of file
+}
diff --git a/packages/SystemUI/src/com/android/systemui/statusbar/phone/StatusBar.java b/packages/SystemUI/src/com/android/systemui/statusbar/phone/StatusBar.java
index 97c229cba6a..f51f77adacc 100644
--- a/packages/SystemUI/src/com/android/systemui/statusbar/phone/StatusBar.java
+++ b/packages/SystemUI/src/com/android/systemui/statusbar/phone/StatusBar.java
@@ -5234,15 +5234,15 @@ public class StatusBar extends SystemUI implements DemoMode,
     }
 
     public void updateBlurVisibility() {
-        if (!mQSBlurEnabled) return;
-
-        int QSBlurAlpha = Math.round(255.0f * mNotificationPanel.getExpandedFraction());
-
-        if (QSBlurAlpha > 0 && !dataupdated && !mIsKeyguard) {
+        if (!mIsKeyguard) {
             DataUsageView.updateUsage();
             dataupdated = true;
         }
 
+        if (!mQSBlurEnabled) return;
+
+        int QSBlurAlpha = Math.round(255.0f * mNotificationPanel.getExpandedFraction());
+
         if (QSBlurAlpha > 0 && !mIsKeyguard) {
             if (!mQSBlurred) {
                 mQSBlurred = true;
@@ -5256,7 +5256,6 @@ public class StatusBar extends SystemUI implements DemoMode,
         } else if (QSBlurAlpha == 0 || mIsKeyguard) {
             mQSBlurView.setVisibility(View.GONE);
             mQSBlurred = false;
-            dataupdated = false;
         }
     }
 
-- 
2.17.1

